pkg/aws/aws.go

```diff
@@ -148,3 +147,21 @@ func GetConfig(ctx context.Context, config map[string]string) (awsConfig *aws.Co
 	}
 	return &aws.Config{Credentials: creds}, region, nil
 }
+
+func IsAwsCredsValid(ctx context.Context, config map[string]string) (bool, error) {
```

Comment:
It is a conscious decision here to pass `config map[string]string` as input , as opposed to `*aws.Config`.  This design is to minimize kanister calls on caller side. All that is needed to validate creds is the the creds itself, without having to call another kanister `aws` func beforehand (such as `GetConfig()`)